libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -259,6 +345,17 @@ - (void)setAccountPersister:(id<SFUserAccountPersister>) persister {
     }
 }
 
+- (SFOAuthCredentials *)retrieveClientCredentials{
```

Comment:
Can we rename this?  It's misleading to say that it's retrieving credentials.  It's bootstrapping new credentials based on configuration.